OPHD/StructureManager.h

```diff
@@ -121,6 +121,20 @@ class StructureManager
 
 	int getCountInState(Structure::StructureClass structureClass, StructureState state) const;
 
+	template <typename StructureType>
+	unsigned int countInState(StructureState state) const
+	{
+		unsigned int count = 0;
+		for (const auto* structure : getStructures<StructureType>())
+		{
+			if (structure->state() == state)
+			{
+				++count;
+			}
+		}
+		return count;
+	}
```

Comment:
I was right about to comment this is a generic method to count operational structures (of any type), and should be generalized. Glad to see you made that change.

Reply:
Yup. Noticed that and was like yeah I can do better.

Not sure why I changed tense in the commit notes for this one but eh, it's whatever :D